internal/server/templates/index.html

```diff
@@ -13,83 +45,71 @@
 		label {
   			margin: 0.2rem 0;
 		}
-		.data-iframe {
-  			position: relative;
-  			top: 16px; 
-  			width: 98%;
-  			min-width: 768px;
-  			height: calc(100vh - 108px); 
-		}
+		
 	</style>
 </head>
 <body>
-<p>
-<form id="urlForm" action="/extract" method="POST" name="scrape" target="data-iframe">
+<div class="page-container">
+<div class="controls-container">
+<form x-data="formHandler" @submit.prevent="handleSubmit">
 <label for="url">Enter a URL:</label>
 <input type="submit" value="Hit It">
 <input type="url" name="url" id="url" value="https://" size="96" maxlength="200" pattern="https?://.*" required title="URL">
-<select title="URL Type" id="actionSelect" onchange="updateFormAction()">
+<select title="URL Type" x-model="formAction">
```

Comment:
UX Suggestion: Swap the placement of the `<input type="submit">` and `<select>` tag.
 
Having the submit button before the text input and URL type is a bit of an anti-pattern. 

![image](https://github.com/user-attachments/assets/90f11c23-3820-4e09-af7e-982de3906c0e)

Reply:
I'm glad the above worked with the trailing paren!

Reply:
And great suggestion!

Reply:
Made this change, I think it's tight!